app/src/main/java/org/eyeseetea/malariacare/receivers/AlarmPushReceiver.java

```diff
@@ -40,17 +40,7 @@
 
     //TODO: period has to be parameterized
     private static final long SECONDS = 1000;
-
-    public static void cancelPushAlarm(Context context) {
```

Comment:
there are situations where we should cancel the push process as logout action, now cancel push process never is executed and I think is a problem